<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Contacts">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>581</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>579</width>
         <height>409</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>   Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>üîç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Clear_all_pb_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Add_page">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>611</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="box_add_name">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>First name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="box_add_family">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Last name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="box_add_email">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="newcontact_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Contact</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Add_Done_pb">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>0</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Add_Done_pb_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>300</y>
        <width>103</width>
        <height>61</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Main</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Home</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Work</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Office</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fax</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="box_phone_number">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Phone number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_this_number_pb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>641</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add  this number</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionClose_project"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Export project</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import </string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
